src/styles/embeds/sass/components/product.css

```diff
@@ -157,46 +165,46 @@
 }
 
 .shopify-buy__btn-and-quantity {
-  .shopify-buy__quantity {
+  & .shopify-buy__quantity {
     border-right: 0;
     border-top-right-radius: 0;
     border-bottom-right-radius: 0;
-    background: $color-white;
+    background: var(--color-white);
   }
 
-  .shopify-buy__quantity-container {
+  & .shopify-buy__quantity-container {
     display: inline-block;
     vertical-align: top;
   }
 
-  .shopify-buy__btn-wrapper {
+  & .shopify-buy__btn-wrapper {
     display: inline-block;
     vertical-align: top;
     margin: 0;
   }
 }
 
 .shopify-buy__cart-item__quantity-container {
-  margin-top: $gutter + 5;
+  margin-top: calc(var(--gutter) + 5);
```

Comment:
the value autoresolves to the measurement type, but you're right, we should be a little more specific. updating to px